docs/getting-started/troubleshooting.md

```diff
@@ -0,0 +1,42 @@
+# Troubleshooting
+
+This section provides a list of common issues and solutions when working with `bpfman`.
+
+## XDP
+
+### XDP Program Fails to Load
+
+When attempting to load an XDP program and the program fails to load:
+
+```console
+$ sudo bpfman load image --image-url quay.io/bpfman-bytecode/xdp_pass:latest xdp --iface veth92cd99b --priority 100
+Error: status: Aborted, message: "An error occurred. dispatcher attach: `bpf_link_create` failed", details: [], metadata: MetadataMap { headers: {"content-type": "application/grpc", "date": "Tue, 28 Nov 2023 13:37:02 GMT", "content-length": "0"} }
```

Comment:
Just a thought we may want to stop adding documentation, or try and minimize the amount that actually has code output from bpfman just because it's becoming a bit of a maintenance nightmare. 

Like in a perfect world we have perfect error variants on `BpfManError` and a troubleshooting guide that goes with that 🤔

Reply:
Fair enough!